examples/plot.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Create plots for privacy evaluation of "shuffling+flipping"."""
+
+from buffling import snr
```

Comment:
Does it make sense to change the file name from snr.py to signal_to_noise_ratio.py? You can also do from buffling import signal_to_noise_ratio as snr and leave the rest of this file untouched.

Reply:
Aha, it makes total sense. I will make change on it.